guides/release/components/component-arguments-and-html-attributes.md

```diff
@@ -134,15 +124,15 @@ the avatar component now, and they will all end up on the element that has
       <div class="cta-note-heading">Zoey says...</div>
       <div class="cta-note-message">
         <p>
-          <code>...attributes</code> syntax follows the same rules as
-          object-spread syntax in JavaScript. If it appears <em>after</em> an attribute,
```

Comment:
Here too, I don't find the explanation of how it's "just like object-spread syntax", to necessarily aid the general reader. It's possible that's because I'm not working to grok the true mental model that underlies templates here, but I'm not sure we need that at this juncture for folks?